fontMetricsData.js

```diff
@@ -2000,6 +2000,7 @@ export default {
         "57683": [0, 0.12, 0, 0, 0.45],
     },
     "Typewriter-Regular": {
+        "32": [0, 0, 0, 0, 0],
```

Comment:
shouldn't one of these numbers be 0.525?

Reply:
Hmm, probably, although it matches the other space definitions in other fonts. Maybe a separate PR? (I think we're getting lucky because we don't use horizontal metrics yet.)

Reply:
@kevinbarabash @edemaine 
The width reported by `parse_ttfs` are incorrect in:
(i) units; they are not divided by `unitsPerEm`, so their numbers are x1000 larger
(ii) they are calculated by `glyph.xMax - glyph.xMin`, which may differ from actual width(`glyph.width`)

Reply:
Thanks for tracking this down, @ylemkimon!  I fixed this in #42. It seems to fix the space width issue too, though I haven't yet tried the two PRs together.